build/teamcity-roachtest-stress.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+source "$(dirname "${0}")/teamcity-support.sh"
+
+google_credentials="$GOOGLE_EPHEMERAL_CREDENTIALS"
+generate_ssh_key
+log_into_gcloud
+export ROACHPROD_USER=teamcity
```

Comment:
it might be worth doing `export GCE_PROJECT=${GCE_PROJECT-andrei-jepsen}` here to avoid a stress run sucking up the quota needed for the nightlies.

Reply:
Yeah, although maybe we should set up a properly named project for stuff like this instead of using `andrei-jepsen`?

Reply:
Agreed in principle, but that means filing a jira ticket against dev-inf and I wouldn't want to block this PR on that, so you need to put another project in as the default and `andrei-jepsen` is de facto used for that and only that.

Reply:
Makes sense, updated with `andrei-jepsen` and added https://cockroachlabs.atlassian.net/browse/DEVINF-140.